doc/ChangeLog

```diff
@@ -1,4 +1,113 @@
 ===============================================================
+Tag name: ctsm5.2.006
+Originator(s): slevis (Samuel Levis)
+Date: Tue 21 May 2024 02:59:05 PM MDT
+One-line Summary: Update externals to cesm2_3_beta17, remove mct, retire /test/tools
+
+Purpose and description of changes
+----------------------------------
+
+ #2493 update externals to beta17
+ #2294 remove mct but not in Externals.cfg
+ #2279 Retire the /test/tools framework for CESM test system custom tests that do the same thing
+
+ Changes unrelated to the tag's title:
+ #2546 fix error in cam4/cam5 test (ekluzek)
+
+Significant changes to scientifically-supported configurations
+--------------------------------------------------------------
+
+Does this tag change answers significantly for any of the following physics configurations?
+(Details of any changes will be given in the "Answer changes" section below.)
+
+    [Put an [X] in the box for any configuration with significant answer changes.]
+
+[ ] clm6_0
+
+[ ] clm5_1
+
+[ ] clm5_0
+
+[ ] ctsm5_0-nwp
+
+[ ] clm4_5
+
+
+Bugs fixed
+----------
+List of CTSM issues fixed (include CTSM Issue # and description) [one per line]:
+ Fixes #2493 update externals to beta17
+ Fixes #2279 Retire the /test/tools framework for CESM test system custom tests that do the same thing
+ Fixes #2546 fix error in cam4/cam5 test (unrelated)
+
+Notes of particular relevance for users
+---------------------------------------
+Changes to documentation:
+ Remove references to mct and cpl7
+
+Substantial timing or memory changes:
+ Not considered
+
+Notes of particular relevance for developers:
+---------------------------------------------
+Changes to tests or testing:
+ I added a long list of izumi nag debug tests to ExpectedFailures as per
```

Comment:
This comment should be removed since CMEPS was updated to the version that fixed this.